### ISSUE TITLE
addStateHandlers() shouldn't fail if a handler returns unknown state keys

### ISSUE THREAD

**Issue by helixbass** [helixbass/ad-hok#38]
Eg this seems to fail badly at runtime:
```
addStateHandlers({a: 1}, {update: () => () => ({a: 2, b: 3}))
```
Seems like it should probably silently ignore the unknown `b` key in the update object?

**Comment by peterstuart** [comment]
Maybe log a warning?

**Comment by helixbass** [comment]
This gets fixed by #63 (updating `addStateHandlers()` to use `useReducer()`)

> Maybe log a warning?

More often than not it'd probably be unintentional to add new state keys, but there's no reason you couldn't intentionally do that now if you want (as far as I'm aware), so I don't think we should apply an opinionated "policy" of warning when someone does that
